### DISCUSSION TITLE
feat(ui): Dashboard filter and highlight

### DISCUSSION THREAD

**Discussion by lkNGAT** [langfuse/langfuse#853] [Ideas]
### Describe the feature you'd like to request
When having a lot of different score names the graph on the dashboard gets really crowded and the individual lines can not be distinguished anymore.

### Describe the solution you'd like to see
It would be helpful to filter for score names like on the other pages. 
Additionally highlighting a score when clicking on the name in the legend would also be great.

### Additional information
No response

### Contribute
No response

**Comment by marcklingen** [comment]
Great suggestion, I'll move this to Discussions to better track and collaborate on feature requests and suggestions. Would you be interested in contributing to this feature?

**Reply by lkNGAT** [reply]
I will take a look into the code base and see if I could to this

**Reply by khareyash05** [reply]
Hey @marcklingen @lkNGAT Can i work on this issue if this is open?

Also - I can think of one more feature - To add a slider to the graphs

**Reply by lkNGAT** [reply]
Hey @khareyash05 
That would be great.

**Reply by khareyash05** [reply]
https://github.com/langfuse/langfuse/pull/907 Please have a look @lkNGAT @marcklingen
